rust/routee-compass-core/src/util/unit/time_unit.rs

```diff
@@ -3,7 +3,7 @@ use crate::util::serde_ops::string_deserialize;
 use serde::{Deserialize, Serialize};
 use std::str::FromStr;
 
-#[derive(Debug, Serialize, Deserialize, Clone)]
+#[derive(Debug, Serialize, Deserialize, Clone, Copy)]
```

Comment:
the real problem here is that these convert ops should take references to their arguments, addressed in #79

Reply:
Yeah that's a good point